af_lib.c

```diff
@@ -1054,7 +1068,7 @@ void af_lib_loop(af_lib_t *af_lib) {
  * loop() for the operation to complete.
  */
 af_lib_error_t af_lib_get_attribute(af_lib_t *af_lib, const uint16_t attr_id) {
-    uint8_t dummy; // This value isn't actually used.
+    uint8_t dummy = 0; // This value isn't actually used.
```

Comment:
It may not be used, but we still probably shouldn't pass around pointers to uninitialized memory :)